week1/homework/src/server.js

```diff
@@ -10,11 +10,50 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
-  });
+    response.setHeader('Content-Type', 'application/json');
+    switch (request.url) {
+      case '/state':
+        state = 10;
+        let number = JSON.stringify({
+          state: state
+        });
+        response.write(`${number}`);
```

Comment:
You don't need to use a template string. `JSON.stringify` already return a `string`.